src/app/api/content/[...slug]/route.ts

```diff
@@ -69,20 +69,29 @@ export function GET(req: Request, { params: { slug } }: RouteProps) {
     current = flatNavItems[i];
 
     // get the "previous" record link to display (that is an actual link)
+    // skip "/docs/rpc" and "/docs/terminology" when setting previous page
     if (flatNavItems.length >= i - 1) {
       for (let j = i; j > 0; j--) {
-        if (!!flatNavItems[j - 1]?.href) {
+        if (
+          !!flatNavItems[j - 1]?.href &&
```

Comment:
Minor suggestion: to coerce to boolean, you can just use `Boolean()` rather than `not not`, which is considered a little clearer. https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript

Reply:
updated ty!

Reply:
coercing to a boolean has other implications though and can give you other type errors when trying to read/use the field later within the if statement. typescript would complain and tell you it is now a boolean and not a string. making it so you cant read the value anymore

in this case it is fine to do since we are not trying to directly read/use `flatNavItems[j]?.href` value as its original type of string, just be warned lol